charts/tekton-apps/templates/wordpress/argocd_applications.yaml

```diff
@@ -188,23 +187,15 @@ spec:
         {{- end }}
 
         livenessProbe:
-          httpGet:
-            path: /wp-admin/install.php
-            port: 8080
-            scheme: HTTP
           periodSeconds: 120
 
         readinessProbe:
-          enabled: true
```

Comment:
Why it is removed at all? Let's leave these by defaqult and add possibility to pass these valyes with helm chart?

Reply:
return